src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2986,6 +2986,20 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "HealthCertificate_Info_primaryButton" = "Weiter";
 
+/* Health Certificate Export Certificates Information */
+
+"HealthCertificate_ExportCertificatesInfo_Title" = "Zertifikare exportieren";
+
+"HealthCertificate_ExportCertificatesInfo_HeaderImageDescription" = "TBD";
+
+"HealthCertificate_ExportCertificatesInfo_Copy01" = "Sie können in einem Schritt alle in Ihrer App vorhandenen Zertifikate in einem gemeinsamen PDF-Dokument speichern. Auf das PDF-Dokument haben zunächst nur Sie Zugriff. Sie können im Anschluss entscheiden, ob Sie es auf Ihrem Smartphone speichern oder in andere Apps importieren möchten.";
+
+"HealthCertificate_ExportCertificatesInfo_Copy02" = "Beachten Sie, dass das PDF-Dokument sensible Informationen enthält. Wir empfehlen Ihnen, hiermit sorgsam umzugehen und es nur Personen vorzuzeigen, denen Sie vertrauen und die zur Prüfung des Nachweises berechtigt sind.";
```

Comment:
**@UA-Team**


Please, don't use "Wir" in texts.

```suggestion
"HealthCertificate_ExportCertificatesInfo_Copy02" = "Beachten Sie, dass das PDF-Dokument sensible Informationen enthält. Es wird empfohlen, hiermit sorgsam umzugehen und es nur Personen vorzuzeigen, denen Sie vertrauen und die zur Prüfung des Nachweises berechtigt sind.";
```

See also our conversation in https://github.com/corona-warn-app/cwa-app-ios/pull/4273#issuecomment-1055582963 (me & @CV113)

Reply:
@flxschmidt

The texts are certainly the ones the UA-Team has given you, but my comment is a suggestion to change that. Maybe @CV113 / whoever will give the final "UA approve" to this PR could come back to me. Thank you anyways for checking!

Reply:
@KathrinLuecke 

The user does not know who "Wir" is. Is it the RKI, the development team, who is it? Also, looking at the rest of the app, there are no other recommendations with "Wir", or?

Reply:
Ok, your decision. Thanks for taking it into consideration.

Reply:
done ("Es wird...")